src/test/scala/de/zalando/beard/parser/BeardTemplateParserSpec.scala

```diff
@@ -321,129 +321,4 @@ class BeardTemplateParserSpec extends FunSpec with Matchers {
       pending
     }
   }
-
-  describe("from file") {
```

Comment:
you shouldn't remove tests, but fix them. In this case you'll have to replace the White + Texts + NewLine with a bigger Text